stdlib/REPL/src/docview.jl

```diff
@@ -687,12 +687,18 @@ stripmd(x::Markdown.Table) =
 
 # Apropos searches through all available documentation for some string or regex
 """
-    apropos(string)
+    apropos([io::IO], string)
 
 Search through all documentation for a string, ignoring case.
```

Comment:
```suggestion
Search through the documentation for entries that contain the string or regular expression, ignoring case.

Print the matching entries to `io`, defaulting to `stdout`.
```

I don't really think this requires a separately documented method — especially when `contains` doesn't do so!  It's also nice to talk about `io` if we're going to mention it.  I've also changed "all documentation" to simply "the documentation."  In my mind, searching _all_ documentation is what juliahub does.

Reply:
Thanks for the review. I didn't quite follow your suggestion exactly, but I have consolidated into a single docstring, mentioned the `io` argument, and clarified what documentation is searched. Let me know if you think the updated version is clear.